### PR TITLE
Update & standardize code blocks in docs

### PR DESCRIPTION
**Pull request by wjwoodson**

### CODE REVIEW

File: content/docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ that week.
 
 - Confirm that all AWS users have MFA configured. The following should return `[]`
 or the users that don't have MFA enabled:
-```shell
+```bash
```
Comment (jmcarp) [inline content/docs/ops/maintenance-list.md]: It looks like we use `shell`, `sh`, and `bash` a bunch throughout this repo. WDYT about picking one and standardizing?

Comment (LinuxBozo) [reply content/docs/ops/maintenance-list.md]: +1 I always err on the side of `sh` and trying to keep things posix compliant so it's truly portable.

**Review by jmcarp — APPROVED** [review]
🚢

**Comment by brittag** [comment]
Non-blocking comment: suggest also adding a short note to https://github.com/18F/cg-site#18f-cloudgov-website documenting the standard that we're using for this.

**Pull request merged**